pkg/accounts/service/avatar.ts

```diff
@@ -99,6 +99,18 @@ export class AccountAvatarService {
     return await this.avatarRepository.findByID(accountID);
   }
 
+  /**
+   * @description Fetch account avatar image metadata.
+   * @param accountIDs
+   */
+  async fetchByAccountIDs(
+    accountIDs: AccountID[],
```

Comment:
```suggestion
    accountIDs: readonly AccountID[],
```
こちらも `readonly` 指定でよいはずです. `AccountHeaderService` の `fetchByAccountIDs` も同様です.